lib/kube/proxy/sess.go

```diff
@@ -928,24 +949,33 @@ func (s *session) leave(id uuid.UUID) error {
 	s.log.Debugf("No longer tracking participant: %v", party.ID)
 	err := s.tracker.RemoveParticipant(s.forwarder.ctx, party.ID.String())
 	if err != nil {
-		return trace.Wrap(err)
+		errs = append(errs, trace.Wrap(err))
 	}
 
 	err = party.Close()
 	if err != nil {
 		s.log.WithError(err).Error("Error closing party")
-		return trace.Wrap(err)
+		errs = append(errs, trace.Wrap(err))
 	}
 
 	if len(s.parties) == 0 || id == s.initiator {
```

Comment:
@xacrimon @espadolini @zmb3 

In kube, once the initiator leaves the session is closed.
Should I change the behavior to be the same as the one in Server access?

Reply:
I don't think you need to change the behavior of Kube right now, so long as recordings work.

Of course, if a moderator leaves, the session should remain open (though it may be locked).